src/test/java/tests/FormEntryTest.java

```diff
@@ -2,9 +2,7 @@
 
 import application.Application;
 import auth.HqAuth;
-import beans.AnswerQuestionRequestBean;
-import beans.AnswerQuestionResponseBean;
-import beans.CurrentRequestBean;
+import beans.*;
```

Comment:
i know in python `.*` is frowned upon for a variety of reasons, but not sure about java